### PR TITLE
Fix typo in arrays.md

### PR DESCRIPTION
**Pull request by ckant787**
There is a spelling mistake so please see it.

### CODE REVIEW

File: doc/src/manual/arrays.md [hunk 1]
```diff
@@ -801,7 +801,7 @@ elements are stored contiguously in column-major order (see additional notes in
 [Performance Tips](@ref man-performance-tips)). The [`Array`](@ref) type is a specific instance
 of `DenseArray`;  [`Vector`](@ref) and [`Matrix`](@ref) are aliases for the 1-d and 2-d cases.
 Very few operations are implemented specifically for `Array` beyond those that are required
-for all `AbstractArrays`s; much of the array library is implemented in a generic
+for all `AbstractArrays`; much of the array library is implemented in a generic
```
Comment (fredrikekre) [inline doc/src/manual/arrays.md]: I think it should be `` `AbstractArray`s ``

**Review by fredrikekre — APPROVED** [review]
Thanks!

File: doc/src/manual/arrays.md (hunk 1 above)
Comment (ckant787) [reply doc/src/manual/arrays.md]: Yes sir you are right.
I am from India.Can you tell me how I can participate in GSOC.How can I become the part of Julia organization.

Comment (fredrikekre) [reply doc/src/manual/arrays.md]: Please ask questions on https://discourse.julialang.org/.

**Pull request merged**